zebra-consensus/src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub mod chain;
 pub mod checkpoint;
 pub mod mempool;
 pub mod parameters;
-pub mod redjubjub;
+#[allow(dead_code)]
```

Comment:
TIOLI: I'd prefer this to be an inner attribute on the primitives module so its closer to the code it annotates.

Reply:
I put it here so it sticks out more and we'll be less likely to forget to remove it later.

Reply:
```suggestion
#[allow(dead_code)] // Remove this once transaction verification is implemented
```
WDYT? @yaahc

Reply:
sounds good